### PR TITLE
Crud permissions

### PR DESCRIPTION
**Pull request by MatiBrego**

### CODE REVIEW

File: src/main/kotlin/app/permission/controller/PermissionController.kt [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RequestParam
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+@RequestMapping("permission")
+class PermissionController {
+    @Autowired
+    val permissionService: PermissionService? = null
```
Comment (JuanSeZ) [inline src/main/kotlin/app/permission/controller/PermissionController.kt]: ?

File: src/main/kotlin/app/permission/model/dto/CreateSnippetInput.kt [hunk 2]
```diff
@@ -0,0 +1,12 @@
+package app.permission.model.dto
+
+import jakarta.validation.constraints.NotBlank
+
+data class CreateSnippetInput(
+    @field:NotBlank(message = "Name must not be blank")
+    val name: String?,
+    @field:NotBlank(message = "Snippet Key must not be blank")
+    val snippetKey: String?,
```
Comment (JuanSeZ) [inline src/main/kotlin/app/permission/model/dto/CreateSnippetInput.kt:9]: What would the snippet key represent ? The id of the snippet in the bucket ?

**Review by JuanSeZ — APPROVED** [review]
MTGL

File: src/main/kotlin/app/permission/model/dto/CreateSnippetInput.kt (hunk 2 above)
Comment (MatiBrego) [reply src/main/kotlin/app/permission/model/dto/CreateSnippetInput.kt:9]: Yes

**Pull request merged**